sql/branch.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+
+    DROP VIEW current_payment_instructions;
```

Comment:
Hmm, why do we have to rebuild the view here? (It made sense in the second step (https://github.com/gratipay/gratipay.com/pull/4061))